### DISCUSSION TITLE
MONAI Bundles Feedback

### DISCUSSION THREAD

**Discussion by vikashg** [Project-MONAI/MONAI#7118] [General]
Hello all, 
I received the following feedback from Mehul Sampath from Flywheel. 
```
Hi Vikash! 
thanks for connecting with me. 
My name is Mehul Sampat and I work at Flywheel. 

We are a medical imaging startup; we provide a platform for uses to ingest/manage and organize their datasets. 

We do not develop ML models inhouse since there are so many open source repos like MONAI MOdel zoo. 

I noticed you are quick active on MONAI ; I think you wrote this gist ? 
https://gist.github.com/vikashg/060b3b1adef84c4808519f3ccf2b85a8
(it is really helpful). 

I was wondering if we could chat about deploying models in production. we love monai-deploy-app but we dislike monai-bundles very much :) 
```

**Comment by vikashg** [comment]
Adding @wyli, @pdogra89 @ericspod @aylward @KumoLiu @mingxin-zheng, @MMelQin @dbericat  for visibility.

**Reply by MMelQin** [reply]
We'd find out if it is in the management/storage of MONAI Bundles, the consumption of it in MONAI Deploy App SDK and in general, or else (not likely the creation of MONAI Bundles given the Flywheel context). +@IntegratorBrad

**Reply by ericspod** [reply]
We would need more details yes to be able to comment on things. With bundles I can definitely understand if the documentation wasn't sufficient, this is why we're updating things for this version so much.

**Comment by dbericat** [comment]
@mpsampat to expand further
